smokefree/sites/all/themes/smokefreegov/templates/footer.tpl.php

```diff
@@ -127,7 +127,7 @@ class="logobar links"
       <input type="hidden" name="filter"          value="0" />
       <input type="hidden" name="getfields"       value="*" />
       <label for="searchbox-footer"
-             class="hiddenSearch hidden">Search</label>
+             class="hiddenSearch sr-only">Search</label>
```

Comment:
Does the task require these changes? I suspect these changes are a part of another ticket which managed to make it into this branch.

While creating new feature branches, please ensure that other other commits are not committed.

Thanks!